### ISSUE TITLE
[Bug] Issues upgrading to react-native v0.71

### ISSUE THREAD

**Issue by Dajust** [ammarahm-ed/react-native-mmkv-storage#316]
**Describe the bug**
Trying to upgrade to RN v0.71 but keep getting the following error

```bash
FAILURE: Build failed with an exception.

* Where:
Build file '/.../node_modules/react-native-mmkv-storage/android/build.gradle' line: 138

* What went wrong:
A problem occurred evaluating project ':react-native-mmkv-storage'.
> Expected directory '/.../node_modules/react-native-mmkv-storage/android/../../../node_modules/react-native/android' to contain exactly one file, however, it contains no files.
```

**To Reproduce**
1. upgrade to RN v0.71
2. Rebuild react-native android

**Expected behavior**
react-native-mmkv-storage should continue working on the latest version of RN.

**Screenshots**

**Platform Information:**
 - OS: Android
 - React Native Version 0.71
 - Library Version 0.8.0
 - Gradle 7.5.1

**Additional context**

**Comment by petrlz63** [comment]
+1

**Comment by kuraydev** [comment]
I am having the same problem. 
@ammarahm-ed Can you take a look at this?

**Comment by playset** [comment]
+1, how to fix this?

**Comment by japhet247** [comment]
+1

**Comment by kuraydev** [comment]
I believe this is going to resolve this issue:
https://github.com/ammarahm-ed/react-native-mmkv-storage/commit/729a85f616b6ef4ec457e7cc9c8b6fa494d3dc7f

**Comment by Ahmedhamed77** [comment]
> I believe this is going to resolve this issue: [729a85f](https://github.com/ammarahm-ed/react-native-mmkv-storage/commit/729a85f616b6ef4ec457e7cc9c8b6fa494d3dc7f)
 when this will be released?

**Comment by ammarahm-ed** [comment]
I'll make a release soonish. Hopefully today.

**Comment by ammarahm-ed** [comment]
Fixed in 0.9.0.

**Comment by kuraydev** [comment]
Thank you for the fix :) @ammarahm-ed
